src/guides/v2.3/migration/migration-tool-internal-spec.md

```diff
@@ -165,6 +165,42 @@ Change prefix option in case Magento was installed with prefix in database table
 
 Configuration data is accessible via \Migration\Config class.
 
+### Steps available operations {#steps}
+
+|Document|Field|
+|---|---|
+|step|Second level node inside the Steps node. Description of the relevant step.|
+|integrity|Specifies the PHP class responsible for the integrity check. Compares the table field names, types, and other info to verify compatibility between Magento 1 and 2 data structures.|
+|data|Specifies the PHP class responsible for the data check. Transfers the data, table by table from Magento 1 to Magento 2.|
+|volume|Specifies the PHP class responsible for the volume check. Compares the number of records between tables to verify that the transfer was successful.|
+|delta|Specifies the PHP class responsible for the delta check. Transfers the delta from Magento 1 to Magento 2 after the full data migration.|
+
+### Source available operations {#source}
+
+|Document|Field|Required?|
+|---|---|---|
+|name|Database name of the Magento 1 server.|yes|
```

Comment:
These all go in the <database> node within the source and destination nodes. And these are not 'available operations' but really database information attributes.

Reply:
@dobooth  Thanks for the info. I have updated the title for this source and destination section. Please review it and let me know if any changes require.